public/_posts/_newsletters/Yearn-Finance-Newsletter-45/es.md

```diff
@@ -0,0 +1,74 @@
+
+## **Resumen**
+
+- Proveedores de liquidez ALPHA e YFI elegibles para un NFT especial
+- Recordatorio para migrar a V2
+- Cómo hacer swap a una vault de Yearn
+- Los ingresos acumulativos del protocolo alcanzan los 39 millones de dólares
+- Vaults de Yearn
+- Noticias del ecosistema
+
+## **Proveedores de liquidez ALPHA e YFI elegibles para un NFT especial**
+
+![](/_posts/_newsletters/Yearn-Finance-Newsletter-45/image2.jpg?w=1456&h=1729)
+
+Los proveedores de liquidez para los fondos Uniswap ETH/YFI y Sushiswap ETH/YFI en los fondos Alpha Homora V2 serán elegibles para una entrega especial de NFT cortesía de Yearn, Alpha y Project Galaxy.
+
+Esta entrega durará 3 días completos, del 3 al 9 de septiembre. Los usuarios que abran o tengan posiciones existentes en grupos elegibles podrán reclamar su NFT en el sitio web de Project Galaxy [aquí](https://galaxy.eco/AlphaFinanceLab/campaign/117).
+
+Tenga en cuenta que los usuarios solo podrán reclamar su NFT durante la ventana y deben reclamar con la dirección de billetera que tiene las posiciones actualmente abiertas
+
+Ver más detalles [aquí](https://twitter.com/AlphaFinanceLab/status/1433689307152195591).
+
+## **Recordatorio para migrar a V2**
+
+La versión 2 ha estado disponible durante meses, las vaults de la v1 han quedado obsoletas y las estrategias se han cerrado, por lo que los usuarios aún en las vaults no obtienen rendimiento. Estas vaults han servido a la comunidad DeFi durante un año como una fuente confiable de rendimiento, y ahora es el momento de una actualización. Como recordatorio, nuestras vaults v2 implican varios cambios, como el despliegue de capital en varias estrategias para una mayor eficiencia y un mejor rendimiento ajustado al riesgo. Puedes migrar a v2 en cualquier momento sin cargos.
+Echa un vistazo a las vaults aquí en [yearn.finance/vaults](https://yearn.finance/vaults).
+
+## **Cómo hacer swap a una vault de Yearn**
+
+![](/_posts/_newsletters/Yearn-Finance-Newsletter-45/image3.jpg?w=679&h=388)
+
+Una de las mejores maneras de intercambiar fondos de una vault de Yearn a otra es usar [curve.fi](https://curve.fi/) o con zaps integrados en la interfaz de usuario de Yearn. Esto es eficiente en el consumo de gas, pero puede tener un coste adicional en deslizamiento o tarifas adicionales.
+
+Otra forma de migrar fondos de Compound y AAVE a Yearn es con el uso de [bowswap.finance](https://bowswap.finance/). Te permite retirarte del protocolo y depositar en una vault de Yearn, y además muestra los APY de ambos protocolos para comparar y te permite seleccionar múltiples tokens a la vez. ¡Sin cargos adicionales!
+Gracias a [cryptouf](https://twitter.com/cryptouf) por curar esta pequeña guía.
+
+## **Los ingresos acumulativos del protocolo alcanzan los 39 millones de dólares**
```

Comment:
AS-IS: ## **Los ingresos acumulativos del protocolo alcanzan los 39 millones de dólares**
TO-BE: ## **Los ingresos acumulados del protocolo alcanzan los 39 millones de dólares**

Reply:
Done